tool/score/fix_excludes.yaml

```diff
@@ -0,0 +1,17 @@
+# A list of rules to which there are no reasonable auto-fix
+# semantics.
+#
+# Rules in this list will be excluded from scoring.
+
+- avoid_as
+- avoid_catching_errors
+- avoid_classes_with_only_static_members
+- avoid_implementing_value_types
+- avoid_positional_boolean_parameters
+- avoid_returning_null_for_future
```

Comment:
We have an existing fix to add `async` to the function body that we could consider using for this one, but I'm not sure it's worth the effort.

Reply:
Maybe not.  For this pass though I'd like to be a bit more conservative and just rule out things that would be obviously crazy.